frontend/src/features/projects/dispose/projectsActionCreator.ts

```diff
@@ -0,0 +1,13 @@
+import { ProjectActions } from '../../../constants/actionTypes';
+import { handleAxiosResponse } from '../../../utils/utils';
+import * as API from 'constants/API';
+import { ENVIRONMENT } from 'constants/environment';
+import CustomAxios from 'customAxios';
+import { saveProjectStatus } from './ProjectWorkflowSlice';
+
+export const fetchProjectWorkflow = () => (dispatch: Function) => {
```

Comment:
actioncreator using handleAxiosResponse to reduce boilerplate. Doesn't seem that useful to apply to all the other actionCreators if we are re-writing the api routing.